snd2png/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project(snd2png)
+cmake_minimum_required(VERSION 3.3.0)
+add_executable(${PROJECT_NAME} ${PROJECT_NAME}.cpp)
+find_package(OpenCV REQUIRED)
+include(FindPkgConfig)
+pkg_check_modules(FFTW3 fftw3)
+pkg_check_modules(SNDFILE sndfile)
+target_link_libraries(${PROJECT_NAME} ${FFTW3_LIBRARIES} ${OpenCV_LIBS} ${SNDFILE_LIBRARIES})
+
+enable_testing()
+add_test(NAME test_snd2png
+    COMMAND sh -c "./snd2png ${CMAKE_CURRENT_SOURCE_DIR}/aplay-captured.wav test.png && md5sum test.png > test.png.md5 && diff -u test.png.md5 ${CMAKE_CURRENT_SOURCE_DIR}/test.png.md5.original")
```

Comment:
you can surely format this more nicely than that

Reply:
done, extracted a test script which is called instead.